i18n/en-US.properties

```diff
@@ -1340,5 +1340,3 @@ boxui.versionHistoryModal.versionLimitExceeded = You can restore the most recent
 boxui.versionHistoryModal.versionNumberBadge = V{versionNumber}
 # Label given to the version badge for screen readers
 boxui.versionHistoryModal.versionNumberLabel = Version number {versionNumber}
-# y
```

Comment:
I'm not sure what this change is about. I think `master` may have had some message with this pattern, which you can see is not part of my commit.

Reply:
It slipped in with the FormattedCompMessage PR. Removing it should be fine. FYI @ehoogerbeets